src/app/rundown/components/scrollable-timeline/scrollable-timeline.component.ts

```diff
@@ -0,0 +1,72 @@
+import {
+  Component,
+  EventEmitter, HostBinding, HostListener,
+  Input,
+  Output,
+} from '@angular/core'
+import { Segment } from '../../../core/models/segment'
+import { PieceLayer } from '../../../shared/enums/piece-layer'
+import { RundownCursor } from '../../../core/models/rundown-cursor'
+import { Part } from '../../../core/models/part'
+import { PartEntityService } from '../../../core/services/models/part-entity.service'
+import { RundownService } from '../../../core/abstractions/rundown.service'
+
+@Component({
+  selector: 'sofie-scrollable-timeline',
+  templateUrl: './scrollable-timeline.component.html',
+  styleUrls: ['./scrollable-timeline.component.scss'],
+})
+export class ScrollableTimelineComponent {
+  @Input()
+  public segment: Segment
+
+  @Input()
+  public pieceLayers: PieceLayer[]
+
+  @Output()
+  public setNextEvent: EventEmitter<RundownCursor> = new EventEmitter()
+
+  public pixelsPerSecond: number = 50
+  public scrollOffsetInMs: number = 0
+
+  public horizontalDragStartPoint?: number
+
+  @HostListener('mousedown', ['$event'])
+  public onDragStart(event: DragEvent): void {
+    this.horizontalDragStartPoint = event.clientX
+    const onDragMove: (event: MouseEvent) => void = this.onDragMove.bind(this)
+    window.addEventListener('mousemove', onDragMove)
+    window.addEventListener('mouseup', () => {
+      this.horizontalDragStartPoint = undefined
+      window.removeEventListener('mousemove', onDragMove)
+    }, { once: true })
+  }
```

Comment:
Just curios, does this also support touch? 
My guess would be yes, but i don't know.

If it doesn't then shouldn't it?

Reply:
Nicely spotted! I don't think it will work with touch, but I don't think it is something that is important.